l10n_be_cooperator/models/tax_shelter_certificate.py

```diff
@@ -169,19 +169,13 @@ def send_certificates(self):
             "l10n_be_cooperator.email_template_tax_shelter_certificate"
         )
         for certificate in self:
-            if (
-                certificate.total_amount_eligible
-                + certificate.total_amount_eligible_previously_subscribed
-                > 0
-            ):
+            if certificate.state == "validated":
```

Comment:
instead of an `if` inside a `for`, this can work too (and is more explicit):
```python
for certificate in self.filtered(lambda x: x.state == "validated"):
```

Reply:
done !